lusine.sargsyan/repo/Lesson_3/exercise_11/exercise_11.py

```diff
@@ -0,0 +1,6 @@
+#! /usr/bin/env python3 
+
+lenght = int(input("Enter the lenght of the field (feets): "))
+width = int(input("Enter the width of the field (feets): "))
+
+print("area is: ", lenght * width / 43560, "acre")
```

Comment:
CSW: What represents the number 43560. Use constant value to hold such kinds of numbers.